lib/ash_json_api/error/error.ex

```diff
@@ -10,7 +10,7 @@ defmodule AshJsonApi.Error do
             meta: :undefined,
             status_code: :undefined,
             internal_description: nil,
-            log_level: :error
+            log_level: :debug
```

Comment:
@zachdaniel there may be consequences that I'm not aware of (like swallowing application exceptions or sth).